aptos-move/transaction-builder-generator/tests/generation.rs

```diff
@@ -231,6 +231,7 @@ test = false
 }
 
 #[test]
+#[ignore]
```

Comment:
?

Reply:
This test unfortunately can't be fixed without the generator supporting generic typings (same exact prob as with aptos_stdlib but with no possible workaround). I can look into that later. Most tests here are already being skipped anyway.

Reply:
leave a comment then